### PR TITLE
Several recipes in one file.

### PR DESCRIPTION
**Pull request by dabrain34**
Cerbero can now detect several recipes in a recipe file
with prefix Recipe...
Can be useful when declaring a demo recipe class.

### CODE REVIEW

File: cerbero/build/cookbook.py [hunk 1]
```diff
@@ -420,21 +420,24 @@ def _load_recipes_from_dir(self, repo):
...
                 continue
-            recipes[recipe.name] = recipe
+            for name in recipes_from_file:
+                recipes[name] = recipes_from_file[name]
         return recipes
 
-    def _load_recipe_from_file(self, filepath, custom=None):
+    def _load_recipes_from_file(self, filepath, custom=None):
+        recipes = {}
         mod_name, file_ext = os.path.splitext(os.path.split(filepath)[-1])
```
Comment (ylatuya) [inline cerbero/build/cookbook.py]: To support correctly multiple recipes, you should move the parsing of the file to the beginning of the function, here, and iterate from here in all Recipe* you find in that dicctionary

Comment (dabrain34) [inline cerbero/build/cookbook.py]: I'm not sure to understand the difference with what i have changed...Do you want me to do the loop earlier as it was done before ?

File: cerbero/build/cookbook.py [hunk 2]
```diff
@@ -451,20 +454,23 @@ def _load_recipe_from_file(self, filepath, custom=None):
...
                         conf.prefix = os.path.join(self._config.prefix, c)
-                r = d['Recipe'](conf)
-                r.__file__ = os.path.abspath(filepath)
-                self._config.arch_config[c].do_setup_env()
-                r.prepare()
-                if self._config.target_arch == Architecture.UNIVERSAL:
-                    recipe.add_recipe(r)
-                else:
-                    return r
+                for key in d:
```
Comment (ylatuya) [inline cerbero/build/cookbook.py]: for key in [x for x in d if x.startswith('Recipe')]

File: cerbero/build/cookbook.py (hunk 1 above)
Comment (turran) [inline cerbero/build/cookbook.py]: I suppose he meant to keep the _load_recipe_from_file parsing method, create a new one named _load_recipes_from_file that will iterate over all the Recipes found on the file and call _load_recipe_from_file there.

**Pull request merged**